src/libstore/globals.hh

```diff
@@ -159,6 +174,18 @@ public:
         )",
         {"build-max-jobs"}};
 
+    MaxSubstitutionJobsSetting maxSubstitutionJobs{
+        this, 10, "max-substitution-jobs",
```

Comment:
That number does not match the description and I think we should use auto as default

Reply:
I would argue that the we should not use the amount of cpus as the default.
One can have a machine with 128 cores or just one but still have 1 Gbit/s bandwidth.
Instead I would set the number of substitution jobs similar to what browsers use.

Reply:
@SuperSandro2000: Oh my, fixed now.

@Mic92: Shall I get rid of `auto` logic (i.e. the `MaxSubstitutionJobsSetting` class) and keep this seeting a simple number for now?

```suggestion
Setting<unsigned int> maxSubstitutionJobs{
        this, 10, "max-substitution-jobs",
```

Reply:
Considering other comments, I removed the `MaxSubstitutionJobsSetting` and used `Setting<unsigned int>` instead. This means there's no `auto` setting anymore.